spec/requests/sessions/login_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'POST api/v1/users/sign_in', type: :request do
+  let(:user) { create(:user) }
+  before { user }
+
+  let(:email) { 'test@test.com' }
```

Comment:
```suggestion
  let(:email) { user.email }
```

This would mean you don't even need to use the `#let!` 😄